test/test_pipeline/components/test_setup_image_augmenter.py

```diff
@@ -0,0 +1,49 @@
+import unittest
+
+from imgaug.augmenters.meta import Augmenter, Sequential
+
+import numpy as np
+
+from autoPyTorch.pipeline.components.setup.augmentation.image.ImageAugmenter import ImageAugmenter
+
+
+class TestImageAugmenter(unittest.TestCase):
+    def test_every_augmenter(self):
+        image_augmenter = ImageAugmenter()
+        configuration = image_augmenter.get_hyperparameter_search_space().sample_configuration()
+        image_augmenter = image_augmenter.set_hyperparameters(configuration=configuration)
+        X = dict(X_train=np.random.randint(0, 255, (8, 3, 16, 16), dtype=np.uint8), image_height=16, image_width=16)
+        for name, augmenter in image_augmenter.available_augmenters.items():
+            if not augmenter.use_augmenter:
```

Comment:
To test every augmenter, there should be a configuration that has `use_augmenter==True` for all augmenters right?